test/lint/lint-submodule.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
```sh
#!/usr/bin/env bash
```

Reply:
Why use bash for it?
sh does the same and is more cross platform